webviews/components/icon.tsx

```diff
@@ -28,3 +28,4 @@ export const diffIcon = <Icon src={require('../../resources/icons/diff.svg')} />
 export const repoIcon = <Icon src={require('../../resources/icons/repo.svg')} />;
 export const gitCompareIcon = <Icon src={require('../../resources/icons/git_compare.svg')} />;
 export const settingsIcon = <Icon src={require('../../resources/icons/settings.svg')} />;
+export const closeIcon = <Icon src={require('../../resources/icons/close.svg')} />;
```

Comment:
This breaks the "Remove" action on the sidebar. Could you add `deleteIcon` as a separate icon since we need both?

<img width="332" alt="CleanShot 2022-10-14 at 16 43 31@2x" src="https://user-images.githubusercontent.com/25163139/195958343-fa06667a-975c-42e6-895b-deeb81a7d42c.png">